Jenkinsfile

```diff
@@ -0,0 +1,186 @@
+#!groovy
+/*
+ * This Jenkinsfile is intended to run on https://ci.jenkins-ci.org and may fail anywhere else.
```

Comment:
So do we really want this here at all? Seems to me that we need [JENKINS-32018](https://issues.jenkins-ci.org/browse/JENKINS-32018).